### PR TITLE
Fix broken routes

### LINKED ISSUE #1249: Add missing routes to Nuxt
#### Current route to be synchronized
- '/rmrk/collections' should show list of collections
- '/rmrk/collection/{id}' shoud show collectionItem

https://github.com/kodadot/nft-gallery/pull/1212#issuecomment-974690288

### PR DESCRIPTION
**Pull request by dezine2dev**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇  _ Do a quick check before the merge. 

### PR type
- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted screenshot of demonstrated change in this PR

### What's new?
- [x] PR closes #1249, #1224
- [ ] Fixed broken routes

### Had issue bounty label ?
- [ ] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=<My_Kusama_Address>)

### Community participation
- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [ ] My fix has changed **something** on UI, a screenshot for others, is best to understand changes.

### CODE REVIEW

File: pages/rmrk/u/_id.vue [hunk 1]
```diff
@@ -73,7 +73,7 @@
           <GalleryCardList
             :items="collections"
             type="collectionDetail"
-            link="rmrk/collection"
+            link="rmrk/collections"
```
Comment (vikiival) [inline pages/rmrk/u/_id.vue:76]: u sure?

**Review by vikiival — COMMENTED** [review]
this does not seem correct (haven't tried)

File: pages/rmrk/u/_id.vue (hunk 1 above)
Comment (dezine2dev) [reply pages/rmrk/u/_id.vue:76]: Good catch. This is my next plan to take a look. 
This is actually not just a problem with this collection.
I see that we need to improve LinkResolver and its usage in nuxt project.

**Comment by yangwao** [comment]
if that fix some routes, but it's not complete fix @dezine2dev

**Pull request merged**